infra/src/Pos/Infra/InjectFail.hs

```diff
@@ -71,14 +71,18 @@ data FInjects m     = FInjects
     , listFInjects   :: m [FInject]
     }
 
+logWarningLoud :: (CanLog m, HasLoggerName m) => String -> m ()
+logWarningLoud s = do
+  logWarning   "***"
+  logWarning $ "*** " <> T.pack s
+  logWarning   "***"
```

Comment:
What about using newlines characters here? What if some other logs got interleaved in-between actually :thinking: ?

Reply:
@KtorZ, done!